SimplenoteUITests/Settings.swift

```diff
@@ -0,0 +1,48 @@
+import XCTest
+
+let screenName = "Settings"
+
+enum SettingsScreen: String {
+
+    case doneButton = "Done"
+    case condensedModeSwitch = "Condensed Note List"
+
+    var element: XCUIElement {
+        switch self {
+        case .doneButton:
+            return app.navigationBars[screenName].buttons[self.rawValue]
+        case .condensedModeSwitch:
+            return app.tables.cells.containing(.staticText, identifier: self.rawValue).firstMatch
+        }
+    }
+}
```

Comment:
Nitpick. When looking at the types names only, it might be hard to guess the difference between `SettingScreen` and `Settings`.

I think this is a good use case for [nested types](https://docs.swift.org/swift-book/LanguageGuide/NestedTypes.html).

```swift
class Settings {

    enum Elements {

        case doneButton
        case condensedModeSwitch
        
        ...
    }

    static func condensedModeEnable() {
        let condensedModeSwitch = Elements.condensedModeSwitch.element
        ...
```

I'm not in love with the `Elements.NAME.element` syntax, but I can't come up with a better name 😳 .

Reply:
This looks better, giving it a try 🙏